features/bootstrap/FeatureContext.php

```diff
@@ -542,17 +548,18 @@ public function install_wp( $subdir = '' ) {
 		$this->proc( 'wp core install', $install_args, $subdir )->run_check();
 	}
 
-	public function install_wp_with_composer() {
+	public function install_wp_with_composer( $vendor_directory = 'vendor' ) {
 		$this->create_run_dir();
 		$this->create_db();
 
 		$yml_path = $this->variables['RUN_DIR'] . "/wp-cli.yml";
 		file_put_contents( $yml_path, 'path: wordpress' );
 
-		$this->proc( 'composer init --name="wp-cli/composer-test" --type="project" --no-interaction' )->run_check();
-		$this->proc( 'composer require johnpbloch/wordpress --optimize-autoloader --no-interaction' )->run_check();
+		$this->composer_command( 'init --name="wp-cli/composer-test" --type="project" --no-interaction' );
+		$this->composer_command( 'config vendor-dir ' . $vendor_directory );
```

Comment:
Something just occurred to me here: We should probably use `escapeshellarg()` on `$vendor_directory`.